src/Code/_variables.scss

```diff
@@ -0,0 +1,17 @@
+
+// Code
+
+$code-font-size:                    87.5% !default;
+$code-color:                        #e83e8c !default;
+
+$kbd-box-shadow:              inset 0 -.1rem 0 rgba($black, .25) !default;
```

Comment:
Nit: I'd love a comment about what kbd stands for

Reply:
👍  will add. (keyboard) I think